### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by sickill**
Running `pip3 install -r requirements.txt` under Python 3 fails:

```Collecting suds-fc==0.4.1 (from -r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/43/b3/bacf8d5f8b9efc5318c539ff3ed3c564a7632a2a55dc58c3508a7551284d/suds-fc-0.4.1.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/3m/q6sc7q4d16x5f8yz_jtmy06r0000gn/T/pip-install-gou_mru0/suds-fc/setup.py", line 20, in <module>
        import suds
      File "/private/var/folders/3m/q6sc7q4d16x5f8yz_jtmy06r0000gn/T/pip-install-gou_mru0/suds-fc/suds/__init__.py", line 181, in <module>
        import client
    ModuleNotFoundError: No module named 'client'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/3m/q6sc7q4d16x5f8yz_jtmy06r0000gn/T/pip-install-gou_mru0/suds-fc/
```

The solution seems to be to upgrade to `suds-jurko`: https://github.com/piratar/safetyvalve/issues/9

### CODE REVIEW

**Comment by sickill** [comment]
After upgrade to `suds-jurko` getting accounts list fails:

```>>> from tfsoffice import Client
>>> api = Client('jenschristian@mailinator.com', 'jij-S4Ss', '825d89ae-5049-4058-bba2-6e29b3728f59')
result = api.accounts.get_taxcode_list()
>>> result = api.accounts.get_taxcode_list()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/resources/accounts.py", line 22, in get_taxcode_list
    return self._client._get_collection(method, None)
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/client.py", line 253, in _get_collection
    results = method(params)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 521, in __call__
    return client.invoke(args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 576, in invoke
    soapenv = binding.get_message(self.method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/binding.py", line 109, in get_message
    content = self.bodycontent(method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/document.py", line 95, in bodycontent
    add_param, self.options().extraArgumentErrors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 83, in parse_args
    return arg_parser(args, kwargs, extra_parameter_errors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 109, in __call__
    return self.__all_parameters_processed()
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 146, in __all_parameters_processed
    self.__check_for_extra_arguments(args_required, args_allowed)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 187, in __check_for_extra_arguments
    self.__error("".join(msg_parts))
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 196, in __error
    raise TypeError("%s() %s" % (self.__method_name, message))
TypeError: GetTaxCodeList() takes 0 positional arguments but 1 was given```

**Comment by sickill** [comment]
On Python 3 getting accounts list fails (work fine on Python 2 with suds-jurko):

```>>> from tfsoffice import Client
>>> api = Client('...', '...', '...')
result = api.accounts.get_taxcode_list()
>>> result = api.accounts.get_taxcode_list()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/resources/accounts.py", line 22, in get_taxcode_list
    return self._client._get_collection(method, None)
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/client.py", line 253, in _get_collection
    results = method(params)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 521, in __call__
    return client.invoke(args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 576, in invoke
    soapenv = binding.get_message(self.method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/binding.py", line 109, in get_message
    content = self.bodycontent(method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/document.py", line 95, in bodycontent
    add_param, self.options().extraArgumentErrors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 83, in parse_args
    return arg_parser(args, kwargs, extra_parameter_errors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 109, in __call__
    return self.__all_parameters_processed()
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 146, in __all_parameters_processed
    self.__check_for_extra_arguments(args_required, args_allowed)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 187, in __check_for_extra_arguments
    self.__error("".join(msg_parts))
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 196, in __error
    raise TypeError("%s() %s" % (self.__method_name, message))
TypeError: GetTaxCodeList() takes 0 positional arguments but 1 was given
```

**Comment by sickill** [comment]
On Python 3 getting accounts list fails (work fine on Python 2 with suds-jurko):

```>>> from tfsoffice import Client
>>> api = Client('...', '...', '...')
result = api.accounts.get_taxcode_list()
>>> result = api.accounts.get_taxcode_list()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/resources/accounts.py", line 22, in get_taxcode_list
    return self._client._get_collection(method, None)
  File "/Users/sickill/projects/vic/python-24so/tfsoffice/client.py", line 253, in _get_collection
    results = method(params)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 521, in __call__
    return client.invoke(args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/client.py", line 576, in invoke
    soapenv = binding.get_message(self.method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/binding.py", line 109, in get_message
    content = self.bodycontent(method, args, kwargs)
  File "/usr/local/lib/python3.7/site-packages/suds/bindings/document.py", line 95, in bodycontent
    add_param, self.options().extraArgumentErrors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 83, in parse_args
    return arg_parser(args, kwargs, extra_parameter_errors)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 109, in __call__
    return self.__all_parameters_processed()
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 146, in __all_parameters_processed
    self.__check_for_extra_arguments(args_required, args_allowed)
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 187, in __check_for_extra_arguments
    self.__error("".join(msg_parts))
  File "/usr/local/lib/python3.7/site-packages/suds/argparser.py", line 196, in __error
    raise TypeError("%s() %s" % (self.__method_name, message))
TypeError: GetTaxCodeList() takes 0 positional arguments but 1 was given
```

**Comment by sickill** [comment]
From @loyning: try this one for py3 : `suds-py3==1.3.3.0`

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-suds-fc==0.4.1
+suds-jurko==0.6
```
Comment (loyning) [inline requirements.txt]: Can you replace suds-jurko with suds-py3==1.3.3.0 ?

suds-jurko was last released in 2014.
https://pypi.org/project/suds-jurko/#history

suds-py3 was updated in 2017.
https://pypi.org/project/suds-py3/#history

**Comment by sickill** [comment]
Great. I see you already upgraded suds. Merging the rest then.

**Pull request merged**